src/NuGet.Clients/NuGet.VisualStudio.Implementation/SolutionExplorer/PackageReferenceAttachedCollectionSourceProvider.cs

```diff
@@ -14,6 +14,7 @@
 
 namespace NuGet.VisualStudio.SolutionExplorer
 {
+    [Microsoft.VisualStudio.ProjectSystem.AppliesTo(ProjectCapability.DependenciesTree)]
```

Comment:
While it doesn't matter from an implementation standpoint today, I think it makes much more sense to use the AppliesToProject attribute given you are already using Order/Name. Especially as they might had default metadata in the future.

Reply:
Should CPS obsolete `AppliesToAttribute` then and use `AppliesToProjectAttribute` instead?

Reply:
I think it does makes sense to have one version of the attribute, but Utilities is a higher layer than CPS.